docs/faq.rst

```diff
@@ -1,33 +1,6 @@
 Frequently asked questions
 ==========================
 
-.. _faq-howto-handle-deprecation-warning:
-
-I cannot suppress the deprecation warning that ``cryptography`` emits on import
```

Comment:
Hmm, we'll emit other warnings and adding/removing this is a pain, I think we probably want to hold on to this, but maybe move it to the bottom?

Reply:
Agreed. This will arise again for as long as the built-in deprecation warning is no used.